.github/workflows/release.yml

```
@@ -0,0 +1,57 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "*"
```

We probably don’t want to release on every tag. Can we configure it to only trigger releases for tags that start with ‘v’, like ‘v1.0.0’? I think that’s the more standard way to set this up.
```suggestion
on:
  push:
    tags:
      - 'v*'
```

---

Good idea, also could be `v*.*.*` to allow only semantic versioned releases.

(Not 100% sure about the this mask)